LIMS2DB/classes.py

```diff
@@ -450,6 +453,8 @@ def get_project_level(self):
         if self.project.udf_dict.get("Reference genome"):
             self.obj['reference_genome'] = self.project.udf_dict.get("Reference genome")
         self.obj['details'] = self.make_normalized_dict(self.project.udf_dict)
+        #Until the Running Notes udf is removed from lims_step
```

Comment:
I guess it will be present for old projects for a long time still?

Reply:
Removed comment!